jersey/src/main/java/com/netflix/conductor/server/resources/WorkflowResource.java

```diff
@@ -114,10 +114,19 @@ public Response startWorkflow(StartWorkflowRequest request, @Context HttpHeaders
 		if (StringUtils.isNotEmpty(correlationId)) {
 			request.setCorrelationId(correlationId);
 		}
-
+		Map<String, Object> authContext=null;
+		if(headers.getRequestHeader("Authorization-Context")!= null) {
+				authContext = executor.validateAuthContext(headers);
+		}
 		NDC.push("rest-start-"+ UUID.randomUUID().toString());
 		try {
-			executor.startWorkflow(workflowId, def.getName(), def.getVersion(), request.getCorrelationId(), request.getInput(), null, request.getTaskToDomain(), auth);
+			if(authContext!=null && !authContext.isEmpty()) {
+				executor.startWorkflow(workflowId, def.getName(), def.getVersion(), request.getCorrelationId(), request.getInput(), null, request.getTaskToDomain(), auth,authContext);
+			}
+			else
```

Comment:
Same style comment as elsewhere.

Reply:
If `authContext` is null or empty is there any harm in calling the `startWorkflow(...)` method that includes the final `authContext` parameter and just getting rid of this if-else statement altogether?

Reply:
@heajuenhwang  @hueys  I updated the code to remove if else statement and call startWorkflow(...) 
 with final authContext param